### DISCUSSION TITLE
How to get value of xcom in ShortCircuitOperator ?

### DISCUSSION THREAD

**Discussion by Vineet-Sharma29** [apache/airflow#24269] [Q&A]
I am using ShortCircuitOperator to bypass downstream task when a certain condition fails. When the condition is true, then I want to use return value of ShortCircuiteOperator task in downstream task. I referenced this [PR](https://github.com/apache/airflow/pull/20071/files) and ran below code for getting xcom value of ShortCircuitOperator task in other task:-
```python
def sample_script(**context):
    task_instance = context['task_instance']
    value = task_instance.xcom_pull(task_ids='shortcircuit_task', key='return_value')
    print(value)
```
However, when I print `value` variable I get `None`. Can someone share how to get xcom value of ShortCircuitOperator task ? I am using Airflow `2.2.1`

**Comment by potiuk** [comment]
I guess you have a typo something. Did you check if the xcom returnvalue is actually pushed (you can see it in the UI).

**Comment by josh-fell** [comment]
Hi @Vineet-Sharma29, the functionality for the `ShortCircuitOperator` to push an `XCom` isn't available until Airflow 2.3.0. You'll have to upgrade or subclass the operator if you don't want to upgrade Airflow versions.
